Makefile

```diff
@@ -51,6 +51,10 @@ venv: requirements.txt requirements-dev.txt
 install-hooks: venv
 	$(BIN)/pre-commit install -f --install-hooks
 
+.PHONY: lint
+lint:
```

Comment:
yeah when I first tried, it reinstalled the whole venv, then re-installed the hooks. Second time and everytime thereafter it only re-installed the hooks on every invocation so I decided I'd leave the dependency out because I'm not really too familiar with the mechanics of how this works.